samples/snippets/deletes/deletes_snippets.py

```diff
@@ -0,0 +1,167 @@
+#!/usr/bin/env python
+
+# Copyright 2022, Google LLC
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START bigtable_deletes_print]
+from google.cloud import bigtable
+
+# Write your code here.
+# [START_EXCLUDE]
+
+
+# [START bigtable_delete_from_column_sample]
+def delete_from_column_sample(project_id, instance_id, table_id):
+    client = bigtable.Client(project=project_id, admin=True)
+    instance = client.instance(instance_id)
+    table = instance.table(table_id)
+    row = table.row("phone#4c410523#20190501")
+    row.delete_cell(column_family_id="cell_plan", column="data_plan_01gb")
+    row.commit()
+    print(
+        "Successfully deleted column 'data_plan_01gb' from column family 'cell_plan'."
+    )
+    for row in table.read_rows():
```

Comment:
do we want to include these print statements in the sample? or just  up until commit maybe? WDYT?

Reply:
The printouts show that the deleted cell was indeed removed from the output, however I think it's not critical.
We can assert that it ran without errors instead. I will remove the printouts in the next commit.

Reply:
Prints have been removed.